scripts/create-pr.sh

```diff
@@ -75,12 +82,11 @@ if ! git diff-index --quiet HEAD --; then
     echo "Creating pull request to the rancher chart.."
     git status
     hub remote add fork $FORK_REPO
-    git checkout -b $VERSION
```

Comment:
This was creating a new branch from an existing new branch created above. This is not needed.
Refer:
https://travis-ci.org/storageos/charts/jobs/595594872#L303 - new version branch
https://travis-ci.org/storageos/charts/jobs/595594872#L329 - second branch off of the above branch